pydata/templates/pydata/generic_import_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base_nav_fixed.html" %}
```

Comment:
Nah, it's fine the way it is. By the way, if we had decided to move this template, it couldn't have "import" -- because we also have "update from URL" function.